pacman_mirrors/pacman_mirrors.py

```diff
@@ -265,7 +267,7 @@ def generate_mirror_list_common(self):
                   txt.NEWLINE)
         else:
             # modify configuration to use default
-            self.modify_config()  # function do self check
+            self.modify_config(DEFAULT)
```

Comment:
I think the problem is here: why always reset default conf when we have a mirrolist?